inst/@sym/frac.m

```diff
@@ -0,0 +1,47 @@
+%% Copyright (C) 2016, 2016 Colin B. Macdonald and Lagu
+%%
+%% This file is part of OctSymPy.
+%%
+%% OctSymPy is free software; you can redistribute it and/or modify
+%% it under the terms of the GNU General Public License as published
+%% by the Free Software Foundation; either version 3 of the License,
+%% or (at your option) any later version.
+%%
+%% This software is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty
+%% of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+%% the GNU General Public License for more details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @documentencoding UTF-8
+%% @deftypefn  {Function File} {@var{y} =} fix (@var{x})
+%% Symbolic fix function.
+%%
+%% Example:
+%% @example
+%% @group
+%% y = frac(sym(3)/2)
+%%   @result{} y = (sym) 1/2
+%% @end group
+%% @end example
+%%
+%% Note: this file is autogenerated: if you want to edit it, you might
+%% want to make changes to 'generate_functions.py' instead.
+%%
+%% @end deftypefn
+
+%% Author: Colin B. Macdonald
+%% Keywords: symbolic
```

Comment:
Either put your name or delete these two lines.  I'm leaning towards delete them.  I think I will stop adding these to new functions as well.